src/function/table.rs

```diff
@@ -137,14 +195,20 @@ impl Table {
         inputs: &[Value],
         ts: u32,
         on_merge: impl FnOnce(Option<Value>) -> Value,
+        unextractable: bool,
```

Comment:
Giving it a second thought, I think `unextractable` and `subsumed` can also be part of `TupleOutput` and we can simply get rid of `Row` struct. The rationale being TupleOutput is supposed to store everything that is determined by the input columns, that is, (meta-)data that are dependent on the input. This is why both timestamp and output are stored in this struct. 

I think either way, it would be great to make sure that `unextractable` and `subsumed` are treated and organized the same way as `timestamp`, since both are row metadata.

Another nice thing about them being encapsulated within the same abstraction is that we can bit-compress these meta data (e.g., store in a single `u64`).